hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestMetaWithReplicasBasic.java

```diff
@@ -80,6 +83,27 @@ public void testZookeeperNodesForReplicas() throws Exception {
     }
   }
 
+  @Test
+  public void testReplicaCleanup() throws Exception {
+    ZKWatcher zkw = TEST_UTIL.getZooKeeperWatcher();
+    List<String> metaReplicaZnodes = zkw.getMetaReplicaNodes();
+    assertEquals(metaReplicaZnodes.size(), 3);
```

Comment:
The parameter order for assertEquals should be switched.

Reply:
Switched